src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/EventFilterConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.sonyericsson.hudson.plugins.gerrit.trigger.config;
+
+import java.util.List;
+
+/**
+ * Interface for Event filter configuration.
+ * @author Christoffer Cortes, christoffer.cortes.sjowall@ericsson.com
+ */
+public interface EventFilterConfig {
```

Comment:
It doesn't seem like this interface is needed? Is it a leftover from when you had the logic in gerrit-events?

Reply:
It's used in PluginConfig. I saw that getter methods had interface stubs so I made them for mine as well.

Reply:
I decided to remove it if that's ok.